src/screenshot.cpp

```diff
@@ -856,3 +864,83 @@ bool MakeScreenshot(ScreenshotType t, const char *name)
 
 	return ret;
 }
+
+static byte _owner_colours[OWNER_END + 1];
```

Comment:
I'm positive this global variable can be avoided. Surely there's no need to "cache" the colours, they can just be looked up when generating the screenshot